File: src/main/java/fr/tikione/c2e/model/web/Edito.java
package fr.tikione.c2e.model.web;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Edito.
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Edito {
    
    private String title;
    private String authorAndDate;
    private String content;
}

File: src/main/kotlin/fr/tikione/c2e/model/web/Edito.kt
package fr.tikione.c2e.model.web

/**
 * Edito.
 */
class Edito {

    var title: String? = null
    var authorAndDate: String? = null
    var content: String? = null

    constructor() {
    }

    constructor(title: String?, authorAndDate: String?, content: String?) {
        this.title = title
        this.authorAndDate = authorAndDate
        this.content = content
    }
}
